department-of-veterans-affairs/va.gov-cms
# VACMS-8719 remove the outdated login page announcement

## Description

Closes #8719. Removes the announcement block from the login page.

## Testing done

1. removed the block from the block layout, logged out, saw that the block was no longer present on the login page.

## Screenshots
prod:
<img width="734" alt="Screen Shot 2022-04-14 at 11 02 11 AM" src="https://user-images.githubusercontent.com/11279744/163450299-506c0df3-1287-4100-816d-a82808f05aa9.png">

local after removing block:
<img width="1200" alt="Screen Shot 2022-04-14 at 11 02 00 AM" src="https://user-images.githubusercontent.com/11279744/163450323-0a096866-4640-422d-b028-09ce6bc243e6.png">

tugboat:
<img width="807" alt="Screen Shot 2022-04-14 at 11 13 01 AM" src="https://user-images.githubusercontent.com/11279744/163451702-1715b868-853f-419b-bf9e-174aecf8eb2a.png">



## QA steps

- visit the tugboat PR environment, see that the announcement block is no longer present on the login page

### Select Team for PR review

- [ ] `Sitewide crew ` (leave Sitewide unchecked and check the specific team instead)
